java/templates/deployment.yaml

```diff
@@ -50,8 +50,12 @@ spec:
       containers:
       - image: {{ .Values.image }}
         name: {{ template "hmcts.releaseName" . }}
-        {{- if or .Values.environment .Values.secrets }}
+        {{- if or .Values.environment .Values.secrets .Values.azureApplicationInsightsInstrumentKey }}
```

Comment:
i wonder if this if statement should go now?
you've added the key as a default so it should always be set?

Reply:
yes, removed the conditions